e2e/tests/notebook/notebook-writer-actions.spec.ts

```diff
@@ -172,18 +117,48 @@ xdescribe('Workspace WRITER Jupyter notebook action tests', () => {
     const linkDisplayed = await workspaceLink.isVisible();
     expect(linkDisplayed).toBe(true);
 
-    // Verify copied notebook exists in collaborator Workspace.
-    notebookCard = await dataResourceCard.findCard({ name: copyNotebookName, cardType: ResourceCard.Notebook });
-    expect(notebookCard).toBeTruthy();
+    await verifyWriterNotebookPermission(page, notebookCopyName);
 
-    // Notebook actions Rename, Duplicate, Delete and Copy to another Workspace actions are available for click.
-    const copyNotebookCardMenu = await notebookCard.getSnowmanMenu();
-    expect(await copyNotebookCardMenu.isOptionDisabled(MenuOption.Rename)).toBe(false);
-    expect(await copyNotebookCardMenu.isOptionDisabled(MenuOption.Duplicate)).toBe(false);
-    expect(await copyNotebookCardMenu.isOptionDisabled(MenuOption.Delete)).toBe(false);
-    expect(await copyNotebookCardMenu.isOptionDisabled(MenuOption.CopyToAnotherWorkspace)).toBe(false);
-    await notebookCard.clickSnowmanIcon(); // close menu
```

Comment:
Created new function `verifyWriterNotebookPermission()` to remove code duplication.